GLOB sdist-make: /Users/thinkingserious/Workspace/python-http-client/setup.py
py26 inst-nodeps: /Users/thinkingserious/Workspace/python-http-client/.tox/dist/python_http_client-1.1.3.zip
py26 installed: argparse==1.3.0,linecache2==1.0.0,python-http-client==1.1.3,six==1.10.0,traceback2==1.4.0,unittest2==1.1.0,wheel==0.24.0
py26 runtests: PYTHONHASHSEED='2351162266'
py26 runtests: commands[0] | /Users/thinkingserious/Workspace/python-http-client/.tox/py26/bin/unit2 discover -v
ERROR: InvocationError: '/Users/thinkingserious/Workspace/python-http-client/.tox/py26/bin/unit2 discover -v'
py27 inst-nodeps: /Users/thinkingserious/Workspace/python-http-client/.tox/dist/python_http_client-1.1.3.zip
py27 installed: python-http-client==1.1.3,wheel==0.24.0
py27 runtests: PYTHONHASHSEED='2351162266'
py27 runtests: commands[0] | /Users/thinkingserious/Workspace/python-http-client/.tox/py27/bin/python -m unittest discover -v
ERROR: InvocationError: '/Users/thinkingserious/Workspace/python-http-client/.tox/py27/bin/python -m unittest discover -v'
py32 inst-nodeps: /Users/thinkingserious/Workspace/python-http-client/.tox/dist/python_http_client-1.1.3.zip
py32 installed: python-http-client==1.1.3,wheel==0.24.0
py32 runtests: PYTHONHASHSEED='2351162266'
py32 runtests: commands[0] | /Users/thinkingserious/Workspace/python-http-client/.tox/py32/bin/python -m unittest discover -v
ERROR: InvocationError: '/Users/thinkingserious/Workspace/python-http-client/.tox/py32/bin/python -m unittest discover -v'
py33 inst-nodeps: /Users/thinkingserious/Workspace/python-http-client/.tox/dist/python_http_client-1.1.3.zip
py33 installed: python-http-client==1.1.3,wheel==0.24.0
py33 runtests: PYTHONHASHSEED='2351162266'
py33 runtests: commands[0] | /Users/thinkingserious/Workspace/python-http-client/.tox/py33/bin/python -m unittest discover -v
ERROR: InvocationError: '/Users/thinkingserious/Workspace/python-http-client/.tox/py33/bin/python -m unittest discover -v'
py34 inst-nodeps: /Users/thinkingserious/Workspace/python-http-client/.tox/dist/python_http_client-1.1.3.zip
py34 installed: python-http-client==1.1.3,wheel==0.24.0
py34 runtests: PYTHONHASHSEED='2351162266'
py34 runtests: commands[0] | /Users/thinkingserious/Workspace/python-http-client/.tox/py34/bin/python -m unittest discover -v
ERROR: InvocationError: '/Users/thinkingserious/Workspace/python-http-client/.tox/py34/bin/python -m unittest discover -v'
py35 inst-nodeps: /Users/thinkingserious/Workspace/python-http-client/.tox/dist/python_http_client-1.1.3.zip
py35 installed: python-http-client==1.1.3,wheel==0.24.0
py35 runtests: PYTHONHASHSEED='2351162266'
py35 runtests: commands[0] | /Users/thinkingserious/Workspace/python-http-client/.tox/py35/bin/python -m unittest discover -v
ERROR: InvocationError: '/Users/thinkingserious/Workspace/python-http-client/.tox/py35/bin/python -m unittest discover -v'
___________________________________ summary ____________________________________
ERROR:   py26: commands failed
ERROR:   py27: commands failed
ERROR:   py32: commands failed
ERROR:   py33: commands failed
ERROR:   py34: commands failed
ERROR:   py35: commands failed
